brainstormforce/astra
# AST-2149 - fix: Woo Cart Color Icon and Checkout Button options dissapear automa…

…tically

### Description

In WooCommerce cart: Cart and checkout color options automatically gets removed when we click on it or trying to apply any color.

### Screenshots


### Types of changes


 fix: In WooCommerce cart: Cart and checkout color options automatically gets removed when we click on it or trying to apply any color.
fix: Made icon visibility better as per suggestions.



### How has this been tested?

- Open Customizer > Header Builder > Cart > Design.
- Change "Color" under Checkout Button Options or Cart Button.
- Options should be visible.
### Checklist:
